<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STChatDetailClass</class>
 <widget class="QWidget" name="STChatDetailClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STChatDetail</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="spChatDetail">
     <property name="styleSheet">
      <string notr="true">QSplitter#spChatDetail::handle {width:1;background-color:#e3e3e3; }



QScrollBar::vertical
{
    width:4px;
    background:rgb(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:0px;   /*上预留位置*/
    padding-bottom:0px;    /*下预留位置*/
}

/*滚动条中滑块的样式*/
QScrollBar::handle::vertical
{
    width:4px;
    background:rgb(0,0,0,25%);
    border-radius:2px;
    min-height:20px;
}

/*鼠标触及滑块样式*/
QScrollBar::handle::vertical::hover
{
    width:5px;
    background:rgb(0,0,0,50%);
    border-radius:2px;
    min-height:20;
}

/*设置下箭头*/
QScrollBar::add-line:vertical
{
    height:0px;
    subcontrol-position:bottom;
}

/*设置上箭头*/
QScrollBar::sub-line:vertical
{
    height:0px;
    subcontrol-position:top;
}</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widTop" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#widTop{border-top: 1px solid #e3e3e3;background-color: rgb(255, 255, 255);}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="lwChatRecordList">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {border:0px;border-right: 1px solid #e3e3e3;}</string>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widBottom" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>280</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widBottom{border-top: 1px solid #e3e3e3;background-color: rgb(255, 255, 255);}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#widget_3{border-right: 1px solid #e3e3e3;}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pbEmotion">
               <property name="styleSheet">
                <string notr="true">

QPushButton {
	border-image: url(:/STSample/Resources/images/emotion.png);
	width: 20;
	height: 20;
}
QPushButton:hover:!pressed {
	border-image: url(:/STSample/Resources/images/emotion_focus.png);
}
QPushButton:hover:pressed{
	border-image: url(:/STSample/Resources/images/emotion_focus.png);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pbScreenShot">
               <property name="styleSheet">
                <string notr="true">

QPushButton {
	border-image: url(:/STSample/Resources/images/screenshot.png);
	width: 20;
	height: 20;
}
QPushButton:hover:!pressed {
	border-image: url(:/STSample/Resources/images/screenshot_focus.png);
}
QPushButton:hover:pressed{
	border-image: url(:/STSample/Resources/images/screenshot_focus.png);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbScreenShotOption">
               <property name="styleSheet">
                <string notr="true">

QPushButton {
	border-image: url(:/STSample/Resources/images/triangle.png);
	width: 10;
	height: 10;
}
QPushButton:hover:!pressed {
	border-image: url(:/STSample/Resources/images/triangle_focus.png);
}
QPushButton:hover:pressed{
	border-image: url(:/STSample/Resources/images/triangle_focus.png);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbLesson">
            <property name="styleSheet">
             <string notr="true">

QPushButton {
	border-image: url(:/STSample/Resources/images/lesson.png);
	width: 20;
	height: 20;
}
QPushButton:hover:!pressed {
	border-image: url(:/STSample/Resources/images/lesson_focus.png);
}
QPushButton:hover:pressed{
	border-image: url(:/STSample/Resources/images/lesson_focus.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>557</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widChatWrite" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-right: 1px solid #e3e3e3;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="teChatWrite">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑 Light'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="acceptRichText">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-right: 1px solid #e3e3e3;border-bottom: 1px solid #e3e3e3;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>585</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbSendMessage">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:pressed {
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:!pressed {
	
	background-color: rgb(109, 206, 160);
}</string>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>teChatWrite</tabstop>
  <tabstop>pbSendMessage</tabstop>
  <tabstop>pbEmotion</tabstop>
  <tabstop>pbScreenShot</tabstop>
  <tabstop>pbScreenShotOption</tabstop>
  <tabstop>pbLesson</tabstop>
 </tabstops>
 <resources>
  <include location="stsample.qrc"/>
 </resources>
 <connections/>
</ui>
